taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ["license_number"]
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if (len(license_number) == 8 and
+                license_number[:3].isalpha() and
+                license_number[:3].isupper() and
+                license_number[3:].isdigit()):
+            return license_number
+        raise ValidationError("Invalid licence number.")
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
```

Comment:
Code Efficiency: Use `get_user_model()` to get the user model. This is important because if you decide to change the user model in the future, you won't have to change it everywhere in your code. Use `get_user_model()` instead of directly importing the `Driver` model.